src/components/learning-guide/guide.cjsx

```diff
@@ -27,7 +27,7 @@ module.exports = React.createClass
 
   renderWeaker: ->
     # sort sections by current level of understanding
-    sortedSections = _.sortBy(@props.allSections, 'current_level')
+    sortedSections = _.sortBy(@props.allSections, (s) -> s.clue.value )
     # if there are less than 4 sections, use 1/2 of the available ones
     weakStrongCount = Math.min(sortedSections.length / 2, 4)
```

Comment:
Minor: Could the `4` be a prop with default value of 4 or something instead since it's a configuration that may change?  or weakStrongCount could even be a prop that's a function, with the default set to something likek `Math.min(sortedSections.length / 2, 4)`